CONTRIBUTING.rst

```diff
@@ -423,9 +423,9 @@ relevant edits to the changelog, (see ``TODO``) including listing the migrations
 for the release. Click on Release, that's it, the rest is automated.
 
 When creating the release, GitHub will save the release info and create a tag with
-the provided version. The new tag will be seen by Travis, which will then create a
-wheel (using the tag as version number, thanks to our ``setup.py``), and push it
-to PyPI (using the new API tokens). That tag should also trigger a ReadTheDocs
+the provided version. The new tag will be seen by Github Actions, which will then
+create a wheel (using the tag as version number, thanks to our ``setup.py``), and push
+it to PyPI (using the new API tokens). That tag should also trigger a ReadTheDocs
```

Comment:
I don't see the part where we do that ...?

Reply:
It is not implemented yet. This PR is currently in draft (but I wanted the CI to run, so I set is as open). I added a check list of what remains to be done in the PR description.

Reply:
CI doesn't run on Draft PRs ? That's not ideal... Is it something we can control ?

Reply:
Seems it does, yay :tada: